reliability-engineering/pipelines/tasks/concourse-provider-config.yml

```diff
@@ -0,0 +1,37 @@
+platform: linux
+image_resource:
+  type: registry-image
+  source:
+    repository: govsvc/task-toolbox
+    tag: latest
+inputs:
+- name: infra # terraform metadata
+outputs:
+- name: concourse-provider-config
+params:
+  DEPLOYMENT_NAME:
```

Comment:
Is it fine to have empty DEPLOYMENT_NAME value?

Reply:
yeah it will just fail, having an empty param makes it easier to see what params the tasks expects... the params get set from the the point where the task is used in the Job/plan.